kernel/src/device/evdev/file.rs

```diff
@@ -27,14 +28,35 @@ use crate::{
     },
     prelude::*,
     process::signal::{PollHandle, Pollable, Pollee},
+    syscall::ClockId,
     util::{
-        ioctl::RawIoctl,
+        ioctl::{dispatch_ioctl, RawIoctl},
         ring_buffer::{RbConsumer, RbProducer, RingBuffer},
     },
 };
 
 pub(super) const EVDEV_BUFFER_SIZE: usize = 64;
 
+mod ioctl_defs {
+    use aster_input::input_dev::InputId;
+
+    use crate::util::ioctl::{ioc, InData, IoctlEnum, OutData};
+
+    // Reference: <https://elixir.bootlin.com/linux/v6.18/source/include/uapi/linux/input.h>
+
+    pub(super) type SetClockId    = ioc!(EVIOCSCLOCKID, b'E', 0xa0, InData<i32>);
+    pub(super) type GetVersion    = ioc!(EVIOCGVERSION, b'E', 0x01, OutData<i32>);
+    pub(super) type GetId         = ioc!(EVIOCGID,      b'E', 0x02, OutData<InputId>);
+    pub(super) type GetDeviceName = ioc!(EVIOCGNAME,    b'E', 0x06, OutData<[u8]>);
+    pub(super) type GetPhys       = ioc!(EVIOCGPHYS,    b'E', 0x07, OutData<[u8]>);
+    pub(super) type GetUniq       = ioc!(EVIOCGUNIQ,    b'E', 0x08, OutData<[u8]>);
+    pub(super) type GetKeyState   = ioc!(EVIOCGKEY,     b'E', 0x18, OutData<[u8]>);
+    pub(super) type GetLedState   = ioc!(EVIOCGLED,     b'E', 0x19, OutData<[u8]>);
+    pub(super) type GetSwState    = ioc!(EVIOCGSW,      b'E', 0x1B, OutData<[u8]>);
+
+    pub(super) type GetBit = IoctlEnum<b'E', 0x20, 0x1F, OutData<[u8]>>;
```

Comment:
Please mention the Linux name otherwise it's hard to look it up from the reference:
```rust
/// The `EVIOCGBIT` ioctl enum.
/// type GetEventBits = IoctlEnum<b'E', 0x20, 0x1F, OutData<[u8]>>;
```

Besides, the example in `ioctl.rs` uses `GetEventBits`, I think it's clearer so let's use it here as well?